outputs.tf

```diff
@@ -1,15 +1,12 @@
-
 locals {
-  binding = try(google_cloud_run_service_iam_binding.binding[0], null)
-  member  = try(google_cloud_run_service_iam_member.member, null)
-  policy  = try(google_cloud_run_service_iam_policy.policy[0], null)
-
-  iam_output = [local.binding, local.member, local.policy]
-
+  binding          = one(google_cloud_run_service_iam_binding.binding)
+  member           = google_cloud_run_service_iam_member.member
+  policy           = one(google_cloud_run_service_iam_policy.policy)
+  iam_output       = [local.binding, local.member, local.policy]
   iam_output_index = var.policy_bindings != null ? 2 : var.authoritative ? 0 : 1
 }
 
 output "iam" {
-  description = "All attributes of the created 'google_cloud_run_service_iam_binding' or 'google_cloud_run_service_iam_member' or 'google_cloud_run_service_iam_policy' resource according to the mode."
+  description = "All attributes of the created `iam_binding` or `iam_member` or `iam_policy` resource according to the mode."
```

Comment:
why did you strip the resource name?

Reply:
Because that is how it is done in [terraform-google-service-account-iam](https://github.com/mineiros-io/terraform-google-service-account-iam/blob/main/outputs.tf#L12) and [terraform-google-storage-bucket-iam](https://github.com/mineiros-io/terraform-google-storage-bucket-iam/blob/main/outputs.tf#L12)

Reply:
Ok, thanks for the explanation (separately). I will revert this change and then over time, we will have to address this in the other repos.

Reply:
both options would be fine for me tbh.. your choice ;)